utils/CircuitCheck/UnitaryBuilder.h

```diff
@@ -43,11 +43,17 @@ class UnitaryBuilder {
   // Helpers
   //===--------------------------------------------------------------------===//
 
-  unsigned getNextQubit() { return std::log2(matrix.rows()); }
+  unsigned getNextQubit() {
+    // Computes log2
+    return 64 - __builtin_clzll(matrix.rows());
```

Comment:
Are we using special compiler builtin functions? (Just curious.)

Reply:
Generally, I'm not opposed their use. However, here it is unnecessary, so I will revert it back.